mypy.ini

```diff
@@ -0,0 +1,17 @@
+[mypy]
+
+# --strict
+disallow_any_generics = True
+disallow_subclassing_any = True
+disallow_untyped_calls = True
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+disallow_untyped_decorators = True
+no_implicit_optional = True
+warn_redundant_casts = True
+warn_unused_ignores = True
+warn_return_any = True
+implicit_reexport = False
+strict_equality = True
+# --strict end
```

Comment:
Can't we throw this in setup.cfg?

Reply:
that's a good idea. I always love keeping the root directory clean and minimal.